about.html

```diff
@@ -0,0 +1,204 @@
+          </li>
+          <li class="list-primary">
+            <a href="" class="link-primary">Sponsor</a>
+          </li>
+          <li class="list-primary list-news">
+            <a href="" class="link-primary">News</a>
+          </li>
+          <li class="list-primary list-orgname">
+            <a href="" class="link-primary link-orgname"> Erriecha </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
+      <section class="align-section section-primary">
+        <div class="humberger-container mobile-only">
+          <a href="javascrip:void()" class="humbergur-link" id="humbergurid">
+            &#9776;
+            </a>
+          </div>
+        <div class="card-primary">
+          <h1 class="event-motto">
+            <q class="motto-quot">Irreechaa Love,Peace and Unity</q>
+          </h1>
+
+          <h2 class="event-title">CELEBRATING IRREECHAA <br> FESTIVAL 2023</h2>          
+        
+        <p class="about-event">
+          <span>
+            Irreechaa is usually celebrated during the weekend following the Meskel holiday. The festival is observed in many areas of Oromia, but, at the national level, 
+          </span>
+          <span class="desktop-only">
+            it brings millions of Oromos from all over Oromia and non-Oromo visitors from other parts of Ethiopia to the shores of Hora (Lake) Arsadi and, as of 2019, to Hora Finfinee in the capital.
+          </span>
+        </p>
+        <p class="primary-contact">
+          Please contact as per email for  the 2023 Irreechaa Celebration <br> Festival irrechafestival2023@otc.org
+        </p>
+      </div>
+      </section>
+      <section class="align-section logo-section">
+        <h2 class="logo-title logo-part">Celebrating Irreechaa Festival 2023 Logo</h2>
+        <p class="logo-part">
+         <span class="desktop-only">
+          This year we hope to expand our invitation to individuals and organizations including world class Medias to promote this beautiful culture to the rest of the world.
+         </span>
+         <span>
+          Irreechaa is usually celebrated during the weekend following the Meskel holiday
+         </span>
+        </p>
+        <div class="logon-border logo-part"><img src="./assets/images/titleicon.png" class="logo-part-icon"> </div>
+      </section>
+<section class="align-section history-section">
+<h2 class="history-head">See The Past Erriecha Festival</h2>
+<p class="history-desc">
+  Pas year Irreechaa Celebration Festival<br>
+Take a look at the last two Irreechaa Celebration Festival which took place in Finfinnee and Bishooftuu
+
+</p>
+  <div class="history-gallery">
+    <article class="gallery-list" overlay="Erriecha 2020">
+      <img src="./assets/images/festival/group-image-1.png" class="history-image">
+    </article>
+    <article class="gallery-list" overlay="Erriecha 2019">
+  <img src="./assets/images/festival/group-image-2.jpg" class="history-image">
+  </article>
+  </div>
```

Comment:
- [ ] @osmanwako great work implementing the changes requested, however as previously requested kindly ensure the images are centered, currently, the are aligned to the left as shown below, and the left and right margins should be equal when you center it. 
![image](https://user-images.githubusercontent.com/61408860/227576186-a5cecd40-a625-46de-9174-856d31da6f01.png)